packages/yum/test.sh

```diff
@@ -274,10 +275,10 @@ echo "::endgroup::"
 
 
 echo "::group::Upgrade"
-if [ -n "${old_package}" ]; then
-  # TODO: Remove this after we release a new version. Old Mroonga package
-  # requires "which" in rpm/post.sh.
-  sudo ${DNF} install -y which
+if [ "${triggered_ref_type}" == "tag" ]; then
```

Comment:
最近の`[`だと`==`でも動く気がするけど、`[`では`=`なんです。

```suggestion
if [ "${triggered_ref_type}" = "tag" ]; then
```

Reply:
fix: [d2392a2](https://github.com/mroonga/mroonga/pull/876/commits/d2392a25496f687ade466e6e54c8a20eaf127605) すみません。よく見たら他もそうなっていますね。


> 最近の[だと==でも動く気がする

手元で動作確認した感じだと動いていました。

```console
$ cat test.sh
#!/bin/bash

set -exu

triggered_ref_type=$1

if [ "${triggered_ref_type}" == "tag" ]; then
  echo "matched!"
else
  echo "no matched!"
fi
echo "fin"
$ bash test.sh tag
+ triggered_ref_type=tag
+ '[' tag == tag ']'
+ echo 'matched!'
matched!
+ echo fin
fin
```